src/dev/impl/DevToys/Views/Tools/Converters/NumberBaseConverter/BasicNumberBaseConverterControl.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<UserControl
+    x:Class="DevToys.Views.Tools.Converters.NumberBaseConverter.BasicNumberBaseConverterControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:DevToys="using:DevToys"
+    xmlns:converters="using:DevToys.UI.Converters"
+    mc:Ignorable="d"
+    d:DesignHeight="400"
+    d:DesignWidth="600">
+    <UserControl.Resources>
+        <DevToys:LanguageManager x:Key="LanguageManager"/>
+    </UserControl.Resources>
+    
+    <StackPanel>
```

Comment:
Add `Spacing="12"`. Otherwise the spacing isn't good:

![image](https://user-images.githubusercontent.com/3747805/158031582-1739e56f-3555-4593-a7d3-58a5d3226348.png)